integration/src/test/scala/ammonite/integration/BasicTests.scala

```diff
@@ -77,9 +77,7 @@ object BasicTests extends TestSuite{
       // tests a bit more tightly; currently the error just goes to stdout
       // and there's no way to inspect/validate it =/
       'tooFew{
-        val errorMsg = intercept[ShelloutException]{
-          exec('basic/"Args.scala", "3")
-        }.result.err.string
+        val errorMsg = exec('basic/"Args.scala", "3").err.string
```

Comment:
This is fine for debugging, but unless we explicitly want to change the behavior we should not leave this in the final codebase